backend/tests/test_packages.py

```diff
@@ -386,3 +390,189 @@ def test_package_maintainers(self):
         )
         self.assertEqual(200, response.json["code"])
         print("test_package_maintainers passed")
+    
+    def test_successful_rating_submit(self):
+        """
+        Test case to verify the behaviour of the system when a user tries to submit rating to a package successfully.
+
+        Parameters:
+        None
+
+        Returns:
+        None
+
+        Raises:
+        AssertionError: If the response code received from the server is not as expected.
+        """
+        uuid = self.login()
+        self.assertEqual(200, response["code"])
+        response = self.client.get(
+            f"/ratings/{self.test_namespace_data['namespace']}/{self.test_package_data['package_name']}",
+            data={"uuid": uuid,"rating":5},
+        )
+        self.assertEqual(200, response.json["code"])
+        print("test_successful_rating_submit passed")
+
+    def test_unsuccessful_rating_invalid_submit(self):
+        """
+        Test case to verify the behaviour of the system when a user tries to submit invalid rating to a package successfully.
+
+        Parameters:
+        None
+
+        Returns:
+        None
+
+        Raises:
+        AssertionError: If the response code received from the server is not as expected.
+        """
+        uuid = self.login()
```

Comment:
I think you should be getting ``access_token`` and then sending it as a Authorization header to the API.

Reply:
Thanks , I had used uuid auth, switched to jwt.